README.md

```diff
@@ -111,7 +176,31 @@ Run the cache for the RNS Service with
 NODE_ENV=ganache npm run bin -- start --enable rns storage --log=debug
 ```
 
-## 3. RIF Marketplace UI
+## 4. RIF Marketplace Upload Service
+Download and setup the RIF Marketplace Upload Service
+```
+git clone git@github.com:rsksmart/rif-marketplace-upload-service.git
+
+cd rif-marketplace-upload-service
+```
+
+Install the dependencies
+```
+npm i
+```
+
+Create the DB using the following command:
+```
+npm run bin -- db-migration --up
+```
+
+Run Upload Service (connected to previously deployed IPFS node)
+```
+NODE_ENV=development npm run bin start -- --log=debug
+```
+
+
+## 5. RIF Marketplace UI
```

Comment:
We should encourage the user to use `npm ci` at least on this project. It should avoid issues with webpack.

Reply:
ok will try it now. If it works we can go with `ci`